### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

File: libtheora/VITABUILD [hunk 1]
```diff
@@ -2,7 +2,7 @@ pkgname=libtheora
 pkgver=1.1.1
 pkgrel=1
 url="https://www.theora.org/"
-source=("http://downloads.xiph.org/releases/theora/${pkgname}-${pkgver}.tar.bz2")
+source=("http://downloads.xiph.org/releases/theora/$pkgname-$pkgver.tar.bz2")
```
Comment (d3m3vilurr) [inline libtheora/VITABUILD:5]: cool this pr.
thank you updating packages.
just one thing, could you restore braces to keep small patch if they didn't change?
(this file is one of cases)

File: xz/VITABUILD [hunk 2]
```diff
@@ -1,17 +1,17 @@
 pkgname=xz
-pkgver=5.2.3
+pkgver=5.2.5
 pkgrel=1
 url="https://tukaani.org/xz/"
-source=("https://tukaani.org/${pkgname}/${pkgname}-${pkgver}.tar.gz")
-sha256sums=('71928b357d0a09a12a4b4c5fafca8c31c19b0e7d3b8ebb19622e96f26dbf28cb')
+source=("https://tukaani.org/$pkgname/$pkgname-$pkgver.tar.gz")
+sha256sums=('f6f4910fd033078738bd82bfba4f49219d03b17eb0794eb91efbae419f4aba10')
```
Comment (d3m3vilurr) [inline xz/VITABUILD:6]: another cases, this only need to update `pkgver` and `sha256sum`

File: libvorbis/VITABUILD [hunk 3]
```diff
@@ -1,9 +1,9 @@
 pkgname=libvorbis
-pkgver=1.3.6
+pkgver=1.3.7
 pkgrel=1
 url="https://xiph.org/vorbis/"
-source=("http://downloads.xiph.org/releases/vorbis/libvorbis-${pkgver}.tar.gz")
-sha256sums=('6ed40e0241089a42c48604dc00e362beee00036af2d8b3f46338031c9e0351cb')
+source=("http://downloads.xiph.org/releases/vorbis/$pkgname-$pkgver.tar.gz")
```
Comment (d3m3vilurr) [inline libvorbis/VITABUILD:5]: this part is ok :+1:

**Review by d3m3vilurr — CHANGES_REQUESTED** [review]

File: harfbuzz/VITABUILD [hunk 4]
```diff
@@ -1,21 +1,18 @@
...
-}
+url="https://github.com/harfbuzz/harfbuzz"
+source=("https://github.com/harfbuzz/harfbuzz/releases/download/$pkgver/$pkgname-$pkgver.tar.xz")
+sha256sums=('4124f663ec4bf4e294d9cf230668370b4249a48ff34deaf0f06e8fc82d891300')
 
 build() {
-  cd harfbuzz-$pkgver
+  cd $pkgname-$pkgver
   mkdir build && cd build
   cmake .. $CMARGS -DCMAKE_INSTALL_PREFIX=$prefix
```
Comment (isage) [inline harfbuzz/VITABUILD:11]: There are few issues with harfbuzz:
1. Now that icu4c is merged, we can enable icu support in harfbuzz
2. Building with cmake doesn't install pkgconfig config (and, libass, for example, requires that, although, that can be worked around), only automake and meson builds are.

File: libtheora/VITABUILD (hunk 1 above)
Comment (scribam) [reply libtheora/VITABUILD]: Done, let me know if I missed something.

File: harfbuzz/VITABUILD (hunk 4 above)
Comment (scribam) [reply harfbuzz/VITABUILD]: I removed harfbuzz from this PR

**Pull request merged**